### PR TITLE
[tech] Add test linter on XML for NeTEx France

### PR DESCRIPTION
**Pull request by woshilapin**

### CODE REVIEW

**Comment by woshilapin** [comment]
The tests are failing for now but they should be fixed once #525 is merged.

File: README.md [hunk 1]
```diff
@@ -42,6 +42,18 @@ If you want to use [PROJ](https://proj.org/) in your code, you can if you
...
+### Feature `xmllint`
+`transit_model` is capable of exporting NeTEx France format. In the tests, we're
+automatically verifying that the produced files are matching the NeTEx
+specification.  For that, we're using the tool `xmllint` which can be install
+on Debian with the package `libxml2-utils`. Therefore, these tests are run only
+if you activate them.
+
+```bash
+apt install libxml2-utils
+cargo test --features xmllint
```
Comment (ArnaudOggy) [inline README.md]: Perhaps the following commands should be added for this to work.
```
git submodule init
git submodule update
```

File: README.md [hunk 2]
```diff
@@ -42,6 +42,18 @@ If you want to use [PROJ](https://proj.org/) in your code, you can if you
...
 
+### Feature `xmllint`
+`transit_model` is capable of exporting NeTEx France format. In the tests, we're
+automatically verifying that the produced files are matching the NeTEx
+specification.  For that, we're using the tool `xmllint` which can be install
+on Debian with the package `libxml2-utils`. Therefore, these tests are run only
+if you activate them.
+
+```bash
+apt install libxml2-utils
```
Comment (ArnaudOggy) [inline README.md]: Perhaps the following commands should be added for this to work.
```
git submodule init
git submodule update
```

File: README.md (hunk 1 above)
Comment (ArnaudOggy) [inline README.md]: I did not look for the cause, but the command does not pass

File: README.md (hunk 2 above)
Comment (woshilapin) [reply README.md]: Thanks. I added some more documentation for that. See d2e3ee2f0d705988720d3f76f60f9f8ad4bbb45a.

File: README.md (hunk 1 above)
Comment (woshilapin) [reply README.md]: It should work now. I had a missing dependency on `proj` feature. See 87e7c10bc0c4bfe896dd1fb23708be0ef6ae5c7d.

**Pull request merged**